app/templates/components/full-main-page-content.hbs

```diff
@@ -0,0 +1,11 @@
+<h2 section="1" {{action 'openSection'}} class="{{if openSection 'open-section'}}">
+	<div class="section-header-label">
+		{{i18n 'app.view-more'}}
```

Comment:
I think `view more` label doesn't fit perfectly here.

Reply:
Let's talk on daily about it.

Reply:
View more approved by Stanley.